calculateHands.js

```diff
@@ -2,10 +2,7 @@ import { ALL_CARD_STRINGS, RANK_TO_NUMRANK, RANKED_HANDS } from './constants';
 import { cardStringToObj } from './util';
 import { myPokerHand } from './handUtils';
 
-function compareHands(hand1, hand2, board) {
-  var hand1Results = myPokerHand(hand1, board);
-  var hand2Results = myPokerHand(hand2, board);
-
+function compareHands(hand1Results, hand2Results) {
   var diff = RANKED_HANDS[hand1Results.hand] - RANKED_HANDS[hand2Results.hand];
```

Comment:
This probably really won't make much difference, but we could add the number from the RANKED_HANDS lookup to the results object so we don't have to do this lookup over and over for hand1

Reply:
Decided not to do to this for now